build.gradle

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-  ext.kotlin_version = '1.4.21'
+  ext.kotlin_version = '1.4.32'
```

Comment:
we can't quite upgrade to 1.5 yet as it has issues with lint - need to wait for the new android studio version for this

Reply:
Does the app uses the same version?

Reply:
yep!